product_docs/docs/biganimal/release/migration/dha_bulk_migration.mdx

```diff
@@ -1,41 +1,40 @@
 ---
 title: Bulk loading data into DHA/PGD clusters
 navTITLE: Bulk loading into DHA/PGD clusters
-description: This guide is specifically for environments where there is no direct access to the PGD Nodes, only PGD Proxy endpoints, such as BigAnimal’s Distributed High Availability deployments of PGD.
+description: This content is specifically for environments where there's no direct access to the PGD nodes, only PGD Proxy endpoints, such as BigAnimal's distributed high availability deployments of PGD.
 deepToC: true
 ---
 
-## Bulk loading data into PGD clusters.
+## Bulk loading data into PGD clusters
 
-**This guide is specifically for environments where there is no direct access to the PGD Nodes, only PGD Proxy endpoints, such as BigAnimal’s Distributed High Availability deployments of PGD.**
+**This content is specifically for environments where there's no direct access to the PGD nodes, only PGD Proxy endpoints, such as BigAnimal's distributed high availability deployments of PGD.**
 
-Bulk loading data into a PGD cluster can, without care, cause a lot of replication load on a cluster. With that in mind, this document lays out a process to mitigate that replication load. 
+Without using care, bulk loading data into a PGD cluster can cause a lot of replication load on a cluster. With that in mind, this content describes a process to mitigate that replication load. 
 
-## Provision or Prepare a PGD Cluster 
+## Provision or prepare a PGD cluster 
 
-You will need to have provisioned a PGD Cluster, either manually, via TPA or on BigAnimal. This will be the target database for
+You must provision a PGD cluster, either manually, using TPA, or on BigAnimal.
```

Comment:
```suggestion
You must provision a PGD cluster, either manually, using TPA, or on BigAnimal. This will be the target database for the migration. Ensure that you provision it with sufficient storage capacity to hold the migrated data.
```